wiremock/src/main/java/com/cognifide/aem/stubs/wiremock/WireMockStubs.java

```diff
@@ -206,10 +208,10 @@ public Config getConfig() {
 
     @AttributeDefinition(
       name = "Global Template Transformer",
-      description = "Enables Pebble template engine / templating"
+      description = "Enables template engine / templating. Handlebars and Pebbles engines are supported"
         + " for response body content and file paths when loading body files. Effectively enables dynamic file loading"
         + " instead of preloading and simplifies defining stubs.")
-    boolean globalTransformer() default true;
+    TransformerEngine globalTransformer() default HANDLEBARS;
```

Comment:
hmm you want to have a nice switch between these two engines, ok.

Reply:
yes. always better then string ;)